src/box/errcode.h

```diff
@@ -51,12 +80,12 @@ struct errcode_record {
  * Please don't forget to do it!
  */
 
-#define ERROR_CODES(_)					    \
+#define ERROR_CODES(_, v) \
 	/*  0 */_(ER_UNKNOWN,			"Unknown error") \
 	/*  1 */_(ER_ILLEGAL_PARAMS,		"Illegal parameters, %s") \
 	/*  2 */_(ER_MEMORY_ISSUE,		"Failed to allocate %u bytes in %s for %s") \
 	/*  3 */_(ER_TUPLE_FOUND,		"Duplicate key exists in unique index \"%s\" in space \"%s\" with old tuple - %s and new tuple - %s") \
-	/*  4 */_(ER_TUPLE_NOT_FOUND,		"Tuple doesn't exist in index '%s' in space '%s'") \
+	/*  4 */v(ER_TUPLE_NOT_FOUND,		"Tuple doesn't exist", ER_FIELDS({{"index", ERF_STR}, {"space", ERF_STR}})) \
```

Comment:
Can you make the macro `_` optionally take more than two arguments? Take a look at how the `ok` macro is implemented in unit tests:

https://github.com/tarantool/tarantool/blob/f90f53900d9a2e4273411d05d8062bdeb9be2415/test/unit/unit.h#L95-L119